### ISSUE TITLE
Structural Activity Relationship Analysis of biological evaluation For Fenarimol Analogues/Epichem compounds

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#51]
Following my matching up structures with their _in vitro/in vivo_ data and LogD data, this is an update on my preliminary SAR for our compounds so far.

I tried to further categorise our 167 screened Fen analogues compounds. I used the initial _in vitro_ leads from our first scree as a starting point and also consider our most recent _in vivo_ hits.

Hits from initial Fen analogue screen:
![Screen Shot 2021-04-27 at 7 00 20 pm](https://user-images.githubusercontent.com/36176821/118084501-2c3d0f00-b404-11eb-937e-d35b2fc9d0a8.png)

Motifs simplified from most recent _in vivo_ screen:
![Motifs of Invivo leads](https://user-images.githubusercontent.com/36176821/118085116-2ac01680-b405-11eb-8a86-4846a9577633.png)

Top row box is selected _in vivo_ compounds. Of these, compounds A, B, C and D are also our top hits from the initial _in vitro_ screen. A, B and C has favourable _in vivo_ efficacy, and D does not. Compound E is a new _in vivo_ lead

Second row box is me trying to simplify the _in vivo_ leads to motifs so that I can go and look through all the _in vitro_ screened MYOS compounds to compare and contrast. Lead A gives motif 1, Leads B and C give motif 2, Lead D gives motif 3 and Lead E gives motif 4.

Bottom row box is the related motifs I saw when I reviewed all the _in vitro_ screened MYOS compounds

For each of the motifs 1–4, I went and searched for related MYOS compounds that we screened so far. I separated them into LogD <2.5 and LogD >5, as well as Growth @ 100 uM of <20%, between 20–80% and > 80%. I managed to do Motifs 1 and 2, but still am working on motifs 3 and 4

Note: For _in vitro_, smaller number = better, for _in vivo_, larger number is better.

**Comment by fantasy121** [comment]
MOTIF 1
![Di-aryl piperazines](https://user-images.githubusercontent.com/36176821/118086208-ef264c00-b406-11eb-8f2f-bb3b3ac528e2.png)

**Comment by fantasy121** [comment]
MOTIF 2
![Tri-aryl alcohols](https://user-images.githubusercontent.com/36176821/118086819-e41feb80-b407-11eb-931e-a18f21044019.png)

**Comment by fantasy121** [comment]
MOTIFS 3 and 4 Work in Progress. 

But from my categorising so far, we can see emerging patterns of good/bad _in vitro_ compounds that are related to the _in vivo_ leads

**Comment by mattodd** [comment]
OK, great. An important analysis that we'll surely need in the paper.
An outcome here would be a clear set of favoured and disfavoured compounds that we can use in a search of the Epichem library to see if there are any compounds lurking there that we really ought to evaluate.

**Comment by fantasy121** [comment]
MOTIF 3

![Tri-aryl alcohols](https://user-images.githubusercontent.com/36176821/120918106-013d9680-c6f6-11eb-9117-a312c50cb4fa.png)

**Comment by fantasy121** [comment]
MOTIF 4

![Like EPL1025:1246](https://user-images.githubusercontent.com/36176821/120918127-1ca8a180-c6f6-11eb-8371-30270a5a639a.png)

**Comment by fantasy121** [comment]
OTHERS

![Others](https://user-images.githubusercontent.com/36176821/120918278-0222f800-c6f7-11eb-82bc-10d3e2b9b28b.png)

**Comment by wwjvdsande** [comment]
@mattodd @fantasy121 @dmitrij176 @MA-Jjingyi @Wilson-Lm Now that the logD values are known I updated the analyses we did on the in vitro and in vivo relationships. Due to the fact that MYOS-00166-00-01 has a LogD of 3.041 it changes our results. Overall you see a trend towards a higher efficacy with a lower logD. However if I split them at 2.5 with Mann-whitney the result becomes negative. May be a different statistical analysis is needed to test if there is still a relation though.

In vivo (not the complete dataset yet, some are still being tested in the larvae):
![vivo](https://user-images.githubusercontent.com/36398138/135468435-bfcbe94c-1479-4ee4-b00a-c3090e11c870.gif)

In vitro, we see more associations with properties of molecules
![Vitro overview](https://user-images.githubusercontent.com/36398138/135469115-75e7b915-230b-4fe2-bc31-4d7a0e9e3d17.gif)

When you split the values into groups it seems that molecules with a logD > 2.5, a mw <400 kDa a flexibility <0.2 and <5 rotation bonds perform better. 
![mann-whitney](https://user-images.githubusercontent.com/36398138/135469758-332099f4-491e-48b4-b4ce-08f2c5fa2c30.gif)

**Comment by fantasy121** [comment]
@wwjvdsande Thanks Wendy. I will see if we can incorporate this new finding into the SAR when we have the SAR meeting this week.

**Comment by fantasy121** [comment]
What do you think of this new layout? @mattodd @dmitrij176 

Numbering of compounds should be updated later when we are certain of compounds ordering (in the draft for the paper) so I'm leaving this numbering as is (with MYOS codes) for now.

I noticed that IC50/IC90 and MIC50/MIC90 are missing for the latest batch of tested compounds. I updated my tables to include IC50 and MIC50, as they help further illustrate the potency of the compounds (and why _in vivo_ were not considered for many of those). @wwjvdsande Are you still working on updating these values for the latest batch?

**Comment by fantasy121** [comment]
I updated the visual summary of SAR Motif A, to have in vitro and in vivo in separate maps. Here's a side-by-side comparison between these two maps, with the new colour-coding patterns.
![SAR Summary Motif A](https://user-images.githubusercontent.com/36176821/139609916-009f6b28-bea7-4ad5-972e-2330d28f392d.png)

**Comment by fantasy121** [comment]
Update of the draft visual summary for Motifs B, C and D, with the new harmonised colour codes. (in vivo coming soon)

![SAR Summary Motif B v2 copy](https://user-images.githubusercontent.com/36176821/140676946-a68b6f7b-a0d4-4953-bab7-a31aed376c58.png)

**Comment by dmitrij176** [comment]
@fantasy121 can you please clarify why does your latest screen includes Motif C when UCL was assigned to cover that part ???

**Comment by dmitrij176** [comment]
![Motif C png](https://user-images.githubusercontent.com/44785501/141192143-bddcc21e-4ad6-43d4-8f27-c77d4ba8eebc.png)

Motif C _in vitro_ analysis. Based on the updated potency guidelines, most compounds are tagged as inactive and there are no more amber analogues in this set. Out of 27 entries, only 3 appear to be potent.

**Comment by mattodd** [comment]
I'm a little confused about what the different "motifs" are here - is there one diagram with them all shown and labelled, so we can compare side by side? We'll need this "legend" for the paper. 

For each figure we're also going to need a way to refer to the compounds therein - so we're going to need MYOS codes under each structure. Not the full code (e.g. "MYOSXXXXAA") but just the Number ("AA"). This also helps to make sure there are no duplicates.

Some of the most useful data arises from matched pairs, which reveal the importance of key groups. These can be just relative changes in potency (clear changes, not minor) rather than necessarily changing a red compound to a green, but still, these are vital. @fantasy121 , I'm assuming that this is what your diagrams show? That when there is a box indicating the alteration of a particular substituent, that all other substituents remain the same, and that that changes you're showing are the only ones? 

@dmitrij176 I would swap columns 3 and 4 of your table since the top pyrimidines should be next to each other (currently columns 2 and 4, should be columns 2 and 3). It's also not immediately clear to me what "R" is - the groups on the left hand side of the table?

Eventually we will need an alignment of styles for all motif/SAR diagrams.

(If you could both (please pretty please) draw your tert-carbon centres correctly? No two bonds should be co-linear).
Throughout all of this, please do bear in mind that the point here is to identify a) trends, b) significant areas of the structures that have not been varied and c) compounds that, were they to be made (or fished out of the Epichem library), would allow us to say more - e.g. the matched pair of a compound where we're trying to figure out the importance of a particular group. @danaklug just did a nice job of this kind of analysis over at OSA opensourceantibiotics/Series-2-Diarylimidazoles#94

**Comment by dmitrij176** [comment]
![Motif C (final)](https://user-images.githubusercontent.com/44785501/142955575-0c394834-4c5a-42c9-bc66-e275572bdca1.png)

Motif C, updated version. 
Legend: each compound number corresponds to the unique MYOS id.

**Comment by dmitrij176** [comment]
Good evening @wwjvdsande. Prior to Christmas holidays, what would be the latest date for UCL to send our new batch of molecules for biological evaluation? As not in terms of season (since its _in vitro_ atm), but for someone at Erasmus to actually collect the delivery before the break? That would be useful to know for synthetic planning as well.

**Comment by wwjvdsande** [comment]
@dmitrij176 if the arrival can be in the week before the christmas holiday it will be fine. Most of us will have two weeks off after christmas.

**Comment by fantasy121** [comment]
@dmitrij176 Could you start looking at match pairs for Motif B next? I will look through A and D for match pairs on my end. Thanks.

**Comment by dmitrij176** [comment]
> @dmitrij176 Could you start looking at match pairs for Motif B next? I will look through A and D for match pairs on my end. Thanks.

Yes Hung. Lets catch up at some point in the future to discuss results.

**Comment by fantasy121** [comment]
![Matched Pairs Motif A copy](https://user-images.githubusercontent.com/36176821/149438861-3a0bd746-fdd2-4412-94c0-c6e28c026d9b.png)
![Matched Pairs Motif A Condensed Vertical](https://user-images.githubusercontent.com/36176821/149438868-8ceb488e-02c8-4c22-bd8f-49e32e2d4210.png)

**Comment by fantasy121** [comment]
Here's my matched pairs for Motif A. Full version (top) and condensed version (bottom). I'm putting together the [SAR writing ](https://docs.google.com/document/d/13iRAEGrZfuupZUADGE5y_iJuicfP5jI0eFRFbVPGOLc/edit)at the moment as well. How are you going with the Matched pairs for B? @dmitrij176 If you want a quick catch-up, let's meet sometime before January meeting?

**Comment by dmitrij176** [comment]
@fantasy121 its almost ready (just need to colour code the structures based on their respective potency). I will upload the scheme on Sunday the latest. Meeting would be useful. What time would suit you?

**Comment by dmitrij176** [comment]
I have a question regarding potency and colour coding: There are a number of compounds in this set which were quite potent at the _in vitro_ stage, but dont have any MIC values. The agreed criteria for amber code is (cited): growth at 25 uM <20% but IC50 and MIC50 >8.0 (mainly dependent on MIC50). In total, 9 compounds do not have MIC values in the Mycetoma database, so its hard to judge by that defined criterion. They are definitely not a red code, since on average most have _in vitro_  performance in the 10-15% range. Any suggestions @fantasy121  @mattodd

**Comment by dmitrij176** [comment]
> Did you check the latest version? We added some MIC50 data a few days ago

Yes. I just rechecked it earlier today before publishing the scheme.

**Comment by dmitrij176** [comment]
To be more specific, its the following entries:
MYOS57, MYOS67, MYOS71, MYOS72, MYOS73, MYOS74, MYOS95, MYOS109, MYOS143

**Comment by wwjvdsande** [comment]
MYOS57, MYOS67, MYOS71, MYOS72, MYOS73, MYOS74, MYOS109 and MYOS143 were only inhibited at 100uM not at 25uM and would therefore not be selected for further testing. After activity at 25uM we will do IC50 and then MIC50. For IC50 and MIC, the highest concentration we test is 16uM. If the compound would not inhibit the fungus at 25uM it is not expected that it would inhibit at a lower concentration. The MIC would then automatically be >16 uM (or even >25 uM, since that is the actual concentation tested of course).

**Comment by dmitrij176** [comment]
![Motif B final (corrected)](https://user-images.githubusercontent.com/44785501/150363150-3b826360-9a69-477f-a99c-cac3d3af4653.png)
Motif B (corrected version)

**Comment by mattodd** [comment]
Hi @dmitrij176 just to clarify - the colour codes are the same as for @fantasy121 's? If so, can each diagram carry the legend, so that the diagrams stand alone and complete? And some compounds have two ID codes, e.g. 43/44 and 1/146 ? @fantasy121 in the January meeting (#67) you presented the picture of what the different motifs are - could you share that here, too? 
It'll be useful to see if some of the SAR conclusions you're seeing are known for cyp51 inhibitors. For example the pyridine to pyrimidine switch (active to inactive). If metal coordination is key from that ring, what's the problem with the pyrimidine? Is that a known feature of other cyp51 inhibitors? Is there a good review/overview anywhere?

**Comment by fantasy121** [comment]
@dmitrij176 So anything that is less than or equal to 20% at 25 uM but not tested for MIC50 will automatically get amber (as Wendy suggested the MIC50 will be >16 by default). (In the excel sheet, you will be the values for 25 uM to be green for these compounds. NB: Excel colours do not corresponds to current legends)

Anything that is greater than 20% at 25 uM and not tested for MIC50 will be given a red. (You will see the values to be amber/black for these compounds)

In this case, all of the compounds you mentioned (57 67 71 72 73 74 109/190 and 143) will all be given red according to the legends because their 25uM activity is above 20%

![Screen Shot 2022-01-27 at 12 22 54 pm](https://user-images.githubusercontent.com/36176821/151274481-f31395ad-d934-4da8-8ac6-f65c8e956fef.png)

**Comment by fantasy121** [comment]
@dmitrij176 Can you also share the raw Chemdraw files for Matched pairing of Motif B and C with me pls? I can make them consistent with the other schemes for the paper if we need to talk about them.

**Comment by fantasy121** [comment]
https://www.dropbox.com/sh/1uyd2rjpgsqty14/AACZ6EUcqF2equgGtd9SdmjZa?dl=0

Excel sheet has been updated with latest in vitro/in vivo results (v2.1 is current, v2 is outdated and now moved to superseded folder)

**Comment by fantasy121** [comment]
![SAR Motif A v1 1 Vertial](https://user-images.githubusercontent.com/36176821/151275375-f7477ff4-7340-484b-ad71-6412e0abacc7.png)

**Comment by fantasy121** [comment]
![Matched Pairs Motif A v1 2 Condensed Vertical](https://user-images.githubusercontent.com/36176821/151275613-4979474b-b690-4675-b2ed-96d46236bbd7.png)
Matched Pairs for G1 Scaffold (Motif A in draft)

**Comment by fantasy121** [comment]
![SAR Summary Motif B v2 2 vertical](https://user-images.githubusercontent.com/36176821/151275880-2890d98a-ac65-44fb-8e16-a0dca7ce896c.png)
G2 scaffold (corresponds to Motifs B, C and D in drafts)

**Comment by dmitrij176** [comment]
> @dmitrij176 Can you also share the raw Chemdraw files for Matched pairing of Motif B and C with me pls? I can make them consistent with the other schemes for the paper if we need to talk about them.

The matched pair files have been uploaded to the Mycetoma Dropbox directory.
https://www.dropbox.com/home/Open%20Source%20Mycetoma

The colour coding was correct in the previous version (see above).

@mattodd coding duplicates have been removed. The original file of each motif was created before I did MYOS coding assignment. Any experimental repeats now have the same MYOS number, I have double checked that.

**Comment by dmitrij176** [comment]
**Motif B**

![Motif B final (corrected)](https://user-images.githubusercontent.com/44785501/151553692-617a9253-4f2e-43cd-baa9-ec927ccf6c9f.png)

**Comment by dmitrij176** [comment]
**Motif C**

![Motif C corrected png](https://user-images.githubusercontent.com/44785501/151553720-d105a403-fb4b-4ae6-9068-395f22201fcf.png)

**Comment by bebi78** [comment]
@fantasy121 @mattodd @MFernflower Do you have some of the active triaryl methanol derivatives left (ca. 50 mg)? I would like to prepare some metal complexes (coordination via the pyridine ring) with these compounds if you agree. I already prepared ruthenium complexes of clotrimazole and bifonazole for my parasite project. There are reports that metal complexes of azoles show improved antifungal and antiparasitic activities when compared with the metal free azole ligands.  Of course, such complexes don't interact with the CYP450 iron center anymore, but they seemingly have other fungal targets, in some cases just a different binding site of the CYP450 protein as to docking calculations.

**Comment by fantasy121** [comment]
![Slide2](https://user-images.githubusercontent.com/36176821/165393787-f8b18564-846d-4b87-bcc8-509f30e2e448.jpeg)
![Slide3](https://user-images.githubusercontent.com/36176821/165393794-08f818ac-9c43-4b70-acfe-2f5326616f1f.jpeg)
![Slide4](https://user-images.githubusercontent.com/36176821/165393798-5ee57953-f6a0-4463-843c-f279a1dd9469.jpeg)
![Slide5](https://user-images.githubusercontent.com/36176821/165393803-d0416d07-f827-4a42-9ba7-768b3e10e6f6.jpeg)

**Comment by fantasy121** [comment]
Updated drafts figures for SAR
